TACC/Core-CMS
# Task/ecep 114 members page and supporting core changes

## Overview

Tweaks for [ECEP Members page](https://prod.ecep.tacc.utexas.edu/alliance-members/) to use manually-created "Show All" link.

## Related

- [ECEP-114](https://jira.tacc.utexas.edu/browse/ECEP-114)
- required by https://github.com/TACC/Core-CMS-Resources/pull/135

## Changes

- support passing manually-created "Show All" link
- update ECEP to use manually-created "Show All" link

## Screenshots

See live demo on [ECEP Members Page (prod)].

## Testing

0. Open [ECEP Members Page (prod)].[^1]
1. Click any state in the nav.
2. Verify the "Show All" link appears beneath nav (narrow and wide screens).
3. Verify "Show All" link still works upon click:
    - Shows all states / page content.
    - Becomes hidden.
4. Verify on narrow screen that sidebar nav links' clickable area extends well beyond text.[^3]

[^1]: <strong>If</strong> you want to test locally, <strong>then</strong> re-create the page [via `djangocms-transfer`](https://github.com/TACC/Core-CMS/wiki/How-to-Use-DjangoCMS-Transfer).[^2]
[^2]: For CMS access, **first** login to ECEP Portal, **then** ping @taoteg or @tacc-wbomar for CMS edit access.
[^3]: This allows fingers to more easily click the links on a mobile device.

## Notes

[build](https://jenkins01.tacc.utexas.edu/job/Core_CMS_Build/611/), [deploy](https://jenkins01.tacc.utexas.edu/job/Core_Portal_Deploy/1389/)

[ECEP Members Page (prod)]: https://prod.ecep.tacc.utexas.edu/alliance-members
